README.md

```diff
@@ -53,41 +80,50 @@ When you ask your assistant to “Turn on the light”, it will use the auth bea
     "automation" :
     {
      "name": "automation",
-     "url": "https://YOUR-URL/openhabGoogleAssistant"
+     "url": "https://us-central1-YOUR-APPLICATION.cloudfunctions.net/openhabGoogleAssistant"
     }
   }
 }
 ```
 
-* Afterwards upload this action file using the following command:
+* Afterwards deploy this action file using the following command:
 ```
-gactions update --action_package action.json --project <YOUR-GOOGLE-CLOUD-PROJECT-ID>
+gactions update --action_package action.json --project <YOUR-APPLICATION>
 ```
 
-Google Assistant will call the service endpoint: `https://YOUR-URL/openhabGoogleAssistant`.
+Google Assistant will call the service endpoint: `"https://us-central1-YOUR-APPLICATION.cloudfunctions.net/openhabGoogleAssistant"`.
```

Comment:
This I also would not hardcode here. This can become quickly obsolete as soon Google changes Google Cloud Functions from Beta to Ready. So please lets keep the generic approach from before. Thx